### PR TITLE
[helm] add option to enable stickness

### PR DESCRIPTION
**Pull request by sherry-x**
add option to enable stickness on devnet fullnode alb

### CODE REVIEW

File: terraform/testnet/testnet/values.yaml [hunk 1]
```diff
@@ -86,6 +86,8 @@ ingress:
   acm_certificate:
   wafAclArn:
   loadBalancerSourceRanges:
+  enableStickyness:
+  cookieDurationSeconds:
```
Comment (rustielin) [inline terraform/testnet/testnet/values.yaml:90]: can we specify a sane default? I believe it's 1 day (86400s) by default on ALB / TF

Comment (sherry-x) [reply terraform/testnet/testnet/values.yaml:90]: sure

**Review by rustielin — APPROVED** [review]
LGTM

**Pull request closed without merging**